qgis_plugin/geoai/core/venv_manager.py

```
@@ -3117,9 +3249,7 @@ def install_dependencies(
                     )
                 return (
                     False,
-                    "Failed to install {}: {}".format(
-                        package_name, install_error_msg[:200]
-                    ),
+                    "Failed to install {}: {}".format(package_name, install_error_msg),
```

Ruff (0.15.21)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)